Assets/MixedRealityToolkit.Examples/Demos/HandTracking/Script/GestureTester.cs

```diff
@@ -70,21 +70,21 @@ public void OnGestureUpdated(InputEventData eventData)
             }
         }
 
-    public void OnGestureUpdated(InputEventData<Vector3> eventData)
-    {
-        Debug.Log($"OnGestureUpdated [{Time.frameCount}]: {eventData.MixedRealityInputAction.Description}");
-
-        var action = eventData.MixedRealityInputAction.Description;
-        if (action == "Manipulate Action")
-        {
-            SetIndicator(ManipulationIndicator, $"Manipulation: updated {eventData.InputData}", ManipulationMaterial, eventData.InputData);
-        }
-        else if (action == "Navigation Action")
+        public void OnGestureUpdated(InputEventData<Vector3> eventData)
```

Comment:
Man, github's whitespace review detection is really bad.

There's actually no delta here aside from whitespace, but what gets rendered here, it looks like so much could have changed.

Reply:
Yeah, i just fixed indentation of these functions, sorry for the noise.